supabase/migrations/20260814095428_app_onboarding_progress.sql

```
@@ -0,0 +1,274 @@
+    (jsonb_typeof(onboarding) = 'object'::text)
+    AND (
+      NOT (onboarding ? 'features'::text)
+      OR (jsonb_typeof((onboarding -> 'features'::text)) = 'object'::text)
+    )
+    AND (
+      NOT (onboarding ? 'setup'::text)
+      OR (
+        (jsonb_typeof((onboarding -> 'setup'::text)) = 'object'::text)
+        AND (
+          NOT ((onboarding -> 'setup'::text) ? 'source'::text)
+          OR (((onboarding -> 'setup'::text) ->> 'source'::text) = ANY (ARRAY['manual'::text, 'cli'::text, 'mcp'::text, 'ai'::text]))
```

SQLFluff (4.2.2)

[error] Line is too long (133 > 80).

(LT05)